src/main/java/org/maxandersen/starfish/Starfish.java

```diff
@@ -168,6 +169,15 @@ public static void editConfig()throws Exception{
 
 }
 
+//Function to Launch the Editor
+public static void launch_editor(Path directory,String ide,String final_clone_path)throws IOException, InterruptedException{
+//If OS is windows then we add .exe after the command    
+ide=System.getProperty("os.name").toLowerCase().indexOf("windows")>=0?ide+".exe":ide;
```

Comment:
hard to read this line. suggestion to clean make it more readable.

```suggestion
boolean isWindows = System.getProperty("os.name").toLowerCase().indexOf("windows")>=0;
ide=isWindows?ide+".exe":ide;
```